### ISSUE TITLE
Investigate using Backstroke to track changes to fork source 

### ISSUE THREAD

**Issue by chrisbolin** [FormidableLabs/react-fast-compare#24]
tip from @marcelmokos:

> Since this is a fork we can use https://backstroke.co/ to track changes in the source project.

**Comment by ryan-roemer** [comment]
We've also diverged from upstream with things like test locations etc. Will need to get a handle on how backstroke handles this.

Also, what the heck is this emoji on `backstroke/server`?

![screen shot 2018-06-05 at 7 47 22 am](https://user-images.githubusercontent.com/445557/40983536-bc8c741a-6894-11e8-918e-a12137adbc6a.png)

**THAT'S NOT BACKSTROKE**

**Comment by chrisbolin** [comment]
yeah, honestly i'd really just like something like `greenkeeper.io`, but `fast-deep-equal` isn't in `dependencies` (nothing is), and there are a lot of things in `devDependencies` which would just create noise. i'll look into this if there's a nice simple config
